api/v1beta1/clusterclass_types.go

```diff
@@ -1072,6 +1086,7 @@ type ClusterClassStatusVariable struct {
 	DefinitionsConflict bool `json:"definitionsConflict"`
 
 	// definitions is a list of definitions for a variable.
+	// +kubebuilder:validation:MaxItems=100
```

Comment:
// +kubebuilder:validation:MaxItems=1000

Shouldn't it be the same max as for .spec.variables?

Reply:
No. This is the number of Runtime Extensions inline that define schemas for the same variable